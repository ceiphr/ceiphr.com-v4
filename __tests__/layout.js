import React from "react"
import renderer from "react-test-renderer"

import Layout from "../src/components/layout"

describe("Layout", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Layout children={<div>Hello World!</div>} location={__PATH_PREFIX__} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})