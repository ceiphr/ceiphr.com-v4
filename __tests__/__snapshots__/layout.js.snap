// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div
  className="container max-w-2xl mx-auto p-6 tk-roboto"
  data-is-root-path={false}
>
  <header
    className="mb-6 no-print"
  >
    <nav
      className="flex justify-between"
    >
      <a
        href="/"
      >
        <p
          className="sr-only"
        >
          Homepage
        </p>
        <test-file-stub
          className="fill-current text-gray-900 dark:text-gray-300"
          height="32"
          width="32"
        />
      </a>
      <ul
        className="flex flex-row text-gray-600 dark:text-gray-400"
      >
        <li
          className="pr-5"
        >
          <a
            className="inline-block mt-1 hover:underline"
            href="/about/"
          >
            About
          </a>
        </li>
        <li
          className="pr-5"
        >
          <a
            className="inline-block mt-1 hover:underline"
            href="https://www.paypal.me/ceiphr"
          >
            Donate
          </a>
        </li>
        <li>
          <a
            className="inline-block mt-1 hover:underline"
            href="https://github.com/ceiphr"
          >
            GitHub
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <main>
    <div>
      Hello World!
    </div>
  </main>
  <footer
    className="mt-6 dark:border-gray-700 text-gray-600 dark:text-gray-400 no-print"
  >
    <p>
      Â© 2021 Ari Birnbaum.
       
      <span
        className="float-right"
      >
        <a
          className="hover:underline"
          href="https://www.iubenda.com/privacy-policy/18781590/legal"
        >
          Privacy
        </a>
      </span>
    </p>
  </footer>
</div>
`;
